core/Rename.cpp

```diff
@@ -96,7 +115,46 @@ namespace olympia
             ev_rename_insts_.schedule();
         }
     }
+    void Rename::getAckFromROB_(const InstPtr & inst_ptr)
+    {
+        sparta_assert(inst_ptr->getStatus() == Inst::Status::RETIRED,
+                        "Get ROB Ack, but the inst hasn't retired yet!");
+
+        core_types::RegFile rf = core_types::RegFile::RF_INTEGER;
+        auto const & dests = inst_ptr->getDestOpInfoList();
+        if(dests.size() > 0){
+            sparta_assert(dests.size() == 1); // we should only have one destination
+            rf = determineRegisterFile(dests[0]);   
+        }
 
+        auto const & dest = inst_ptr->getRenameData().getDestination();
+        auto const & original_dest = inst_ptr->getRenameData().getOriginalDestination();
+        auto const & srcs = inst_ptr->getRenameData().getSource();
+
+        if(dests.size() > 0){
```

Comment:
Duplicate `if` condition.  Can this be combined with the previous `if` statement?

Reply:
Addressed in [3eb203c](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/3eb203c761cd11fbebea435284f6d14ba45437a7)